proto/serving.proto

```diff
@@ -43,6 +43,7 @@ message FeatureServeRequest {
   repeated FeatureID features = 1;
   repeated Entity entities = 2;
   Model model = 3;
+  int32 requestID = 4;
```

Comment:
its an int here but a string everywhere else

Reply:
I actually dont think we need this, not sure when/why I added this